### PR TITLE
Fixes missing trigger transaction props and adds experimental support for concurrent price update processing

### PR DESCRIPTION
**Pull request by drub0y**
- Fixes trigger txns to have explicitly set nonce
- Supports concurrent price update processing

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -1462,9 +1470,14 @@ if(AUTO_HARVEST_MS > 0){
 */
 function createTransaction(additionalTransactionProps, isPriority = false) {
 	const transaction = {
+		common: COMMON_TRANSACTION_PROPS,
```
Comment (0xCrumb) [inline bot.js:1473]: Need to remove common. causing error in conjunction with setting chain in the object.

File: bot.js [hunk 2]
```diff
@@ -76,6 +77,15 @@ if (!NETWORK) {
 	throw new Error(`Invalid chain id: ${CHAIN_ID}`);
 }
 
+const COMMON_TRANSACTION_PROPS = Object.freeze({
```
Comment (0xCrumb) [inline bot.js:80]: Not necessary if we already set chainId in the object.

File: bot.js (hunk 1 above)
Comment (0xCrumb) [reply bot.js:1473]: In fact for speed optimisations we only need to add nonce + chainId, the rest can go - less things to keep track of

**Review by 0xCrumb — CHANGES_REQUESTED** [review]

File: bot.js (hunk 2 above)
Comment (drub0y) [reply bot.js:80]: Needed it for `hardfork`, but remembered you can just put `hardfork` on the transaction itself too.

File: bot.js (hunk 1 above)
Comment (drub0y) [reply bot.js:1473]: Ah, I already had `hardfork` here explicitly. I wonder why I wasn't getting the error you were getting tho. Anywho, this
should be gtg now!

**Comment by 0xCrumb** [comment]
lgtm, ran locally without any problems

**Review by 0xCrumb — APPROVED** [review]
Ran all night without issues. Approved

**Pull request merged**